### ISSUE TITLE
fpack-compressed fits files 

### ISSUE THREAD

**Issue by vertighel** [astrom-tom/dfitspy#14]
Please consider supporting files compressed with `fpack`, as an analogy with `dfits -x 1 *.fits*`

**Comment by astrom-tom** [comment]
Hi @vertighel !
Would you have an example file?

Thanks!

**Comment by vertighel** [comment]
hello,

sure. here it is in attachment.

Cheers,
Davide Ricci




Il giorno mer 27 mag 2020 alle ore 23:25 R. THOMAS <notifications@github.com>
ha scritto:

**Comment by astrom-tom** [comment]
Hi @vertighel 
I can't see the file, would you mind trying to send it again?
Thanks!

**Comment by astrom-tom** [comment]
Hi @vertighel.

I added the possibility to open  *.fz files in the latest version of the module (20.7.1) along other important changes. 
Let me know if this is ok for you!

Cheers.
R.

**Comment by vertighel** [comment]
Hello,

sorry for the late responde.
At the end were you able to open it?

Il giorno ven 24 lug 2020 alle ore 18:09 R. THOMAS <notifications@github.com>
ha scritto:

**Comment by astrom-tom** [comment]
Nope, but I managed to found one on a public archive and could make it work. If you can try the new version and let me know that would be great!
Thanks!
